installer/installation-notes.md

```diff
@@ -0,0 +1,135 @@
+# Installing the RSP
+
+## Preparation
+
+  * Create a working dir.
+  * git clone https://github.com/lsst-sqre/phalanx into it (using https
+    rather than git+ssh is important).
+  * cd phalanx/installer
+
+## Create installation environment
+
+  * If you don't have a virtualenv for the installer already,
+    `mkvirtualenv -p $(which python3) -r requirements.txt installer`
+    (adjust if you use pyenv or raw virtualenv or whatever; the
+    important thing is to pick up the requirements.txt from the
+    installer directory)
+  * Get the correct write token for this instance's Vault enclave out of
+    1Password
+
+## Prepare for the ingress
+
+There are three cases (thus far) for what the top-level ingress looks
+like.
+
+  * It is externally managed, as are any required TLS certificates.
+    The installer doesn't cope with this case very well yet, and you
+    will need to manually inject the TLS certificate into the squareone
+    configuration post-installation.
+
+  * For each of the second and third, which are the cases the installer
+    script is designed to support, the RSP manages the top-level
+    ingress, and Let's Encrypt is used to manage TLS certificates.  The
+    difference between these cases is that one has a pre-defined fixed
+    IP address (e.g. the T&S sites), and one uses whatever the
+    Kubernetes provider gives up (e.g. GKE).
+
+  * In the second case, now is a good time to make sure that the top-level
+    endpoint (e.g. "tucson-teststand.lsst.codes") resolves to the
+    correct IP address, whether directly as an A record or as a CNAME to
+    an A record.
+
+  * For the third case you will set up the DNS record once you know the
+    IP address of the endpoint.
+
+If this is the first time you've installed the RSP in a given domain,
+and you are managing TLS certs via Let's Encrypt for the RSP instance,
+you will need to put a glue record into that domain to allow Let's
+Encrypt to work its magic.  This process is documented at
+https://phalanx.lsst.io/ops/cert-issuer/route53-setup.html for Route
+53.  If you're not using Route 53, you're on your own, but use that
+document as a guide for the tasks you will need to perform.
+
+## Pre-flight check
+
+Check the rest of the phalanx definitions.  Particular items of
+ interest:
```

Comment:
I don't think the reader will know what "phalanx definitions" means. I think you mean the values files? You probably need a step here to create values files for your new deployment if this is a new deployment. There is some (but not very much) documentation in the Phalanx docs already for this that you can link to.